### PR TITLE
Uplift RT1.5 for Routerid, global address family, route-policy

### PR DESCRIPTION
**Pull request by singh-prem**
Uplifted RT1.5 to include:
1. Router ID
2. Global Address family
3. default route-policy to permit all
4. removed skip from verifyPrefixLimitTelemetry and fixed logic to handle v4 and v6 neighbor
5. fixed verifyPortsUp to only verify status of ports being used in the test (2 in count)

### CODE REVIEW

**Comment by singh-prem** [comment]
any update ?

**Review by thesrinath — COMMENTED** [review]
Can you gofmt this file?

**Review by thesrinath — APPROVED** [review]

File: internal/deviations/deviations.go [hunk 1]
```diff
@@ -124,4 +124,6 @@ var (
 	RoutePolicyUnderPeerGroup = flag.Bool("deviation_rpl_under_peergroup", false, "Device requires route-policy configuration under bgp peer-group. Fully-compliant devices should pass with and without this deviation.")
 
 	MissingPrePolicyReceivedRoutes = flag.Bool("deviation_prepolicy_received_routes", false, "Device does not support bgp/neighbors/neighbor/afi-safis/afi-safi/state/prefixes/received-pre-policy. Fully-compliant devices should pass with and without this deviation.")
+
+	RoutePolicyUnderNeighborAfiSafi = flag.Bool("deviation_rpl_under_neighbor_afisafi", false, "Device requires route-policy configuration under bgp neighbor afisafi. Fully-compliant devices should pass with this deviation set to true.")
```
Comment (xw-g) [inline internal/deviations/deviations.go:128]: https://github.com/openconfig/featureprofiles/blob/97c29dc0c8f89dfe80c449bd78d14a2cbb475d4b/feature/bgp/policybase/ate_tests/route_installation_test/route_installation_test.go#L189-L190

Seems like `RoutePolicyUnderNeighborAfiSafi` is a duplicate of `RoutePolicyUnderPeerGroup`? This is related to what I meant in https://github.com/openconfig/featureprofiles/pull/871/files#r1080686087

**Pull request merged**